graphs/rdf-construct.fbp

```diff
@@ -0,0 +1,32 @@
+# rdf-construct.fbp
+
+# Retrieves the source-graph (IRI) from the sparql-endpoint and returns parsed JSON-LD
```

Comment:
We need it in rdf-construct-mocha to test round trip. Without it there is no way to ensure rdf-insert succeeded.